.github/workflows/ci.yml

```diff
@@ -496,26 +496,6 @@ jobs:
         if: matrix.job == 'bench' && !startsWith(github.ref, 'refs/tags/')
         run: cargo bench --locked
 
-      - name: Post Benchmarks
-        if: |
-          matrix.job == 'bench' &&
-          github.repository == 'denoland/deno' &&
-          github.ref == 'refs/heads/main' && !startsWith(github.ref, 'refs/tags/')
-        env:
-          DENOBOT_PAT: ${{ secrets.DENOBOT_PAT }}
-        run: |
-          git clone --depth 1 --branch gh-pages                             \
-              https://${DENOBOT_PAT}@github.com/denoland/benchmark_data.git \
-              gh-pages
-          ./target/release/deno run --allow-all --unstable \
-              ./tools/build_benchmark_jsons.js --release
-          cd gh-pages
-          git config user.email "propelml@gmail.com"
-          git config user.name "denobot"
-          git add .
-          git commit --message "Update benchmarks"
-          git push origin gh-pages
-
```

Comment:
Is it possible for both of these to live simultaneously for some time? How much work would that be. I'd rather not make a hot switch all at once.

Reply:
Added back